signing_service/signing_service/utils.py

```diff
@@ -110,7 +112,11 @@ def __call__(  # type: ignore
             assert False
         if self.base64_convert:
             assert isinstance(self.const, str)
-            self.const = b64decode(self.const)
+            try:
+                self.const = b64decode(self.const)
+            except binascii.Error as exception:
+                logger.error(f'Unable to decode "{self.const}", {exception}')
+                raise Base64DecodeError(f'Unable to decode "{self.const}", {exception}')
```

Comment:
Would it be possible to report the error using the mechanisms built into argparse? What happens if you raise `argparse.ArgumentTypeError` or `ValueError` here?

It should work for [argparse.type](https://docs.python.org/3/library/argparse.html#type), not sure about a class inheriting from `argparse.Action`.

Reply:
I'm not sure what You mean. Raising `argparse.ArgumentTypeError` or `ValueError` in this line instead of `Base64DecodeError` work exactly the same. Same stacktrace with different error at the end.
But if You want me to add `type` checking to `SecretProvider(Action)` class it's going to require some significant refactoring, and will take me a lot of time, that can be spend on something more important.